SerialPrograms/Source/PokemonSV/Programs/ShinyHunting/PokemonSV_ShinyHunt-Scatterbug.cpp

```diff
@@ -270,6 +274,7 @@ void ShinyHuntScatterbug::handle_battles_and_back_to_pokecenter(SingleSwitchProg
                 }
             }catch (ProgramFinishedException&){
                 GO_HOME_WHEN_DONE.run_end_of_program(context);
+                send_program_finished_notification(env, NOTIFICATION_PROGRAM_FINISH);

```

Comment:
I believe this is sent automatically if it's thrown as an exception.

Reply:
updated